### ISSUE TITLE
[BUG] SVG wallpaper appears differently in Safari in comparison to other browsers

### ISSUE THREAD

**Issue by mureev** [excalith/excalith-start-page#74]
### Describe The Bug
SVG wallpaper appears significantly darker in Safari in comparison to other browsers.

### To Reproduce

1. Open [online version of the page](https://excalith-start-page.vercel.app) in Safari on macOS/Mac device or using iPhone

### Expected Behavior
I would expect visually background to look the same in Safari as in Chrome, or to have an option to adjust.

### Screenshots
![ 2024-03-24 в 17 53 19](https://github.com/excalith/excalith-start-page/assets/277148/ed3809d4-b0a4-401a-97c0-1467d1ab1b0d)

![2024-03-24 в 13 47 52](https://github.com/excalith/excalith-start-page/assets/277148/d34e2ae4-cb97-4042-8abf-24a976b6e4bd)

### Specs

- OS: macOS Ventura 13.6 and iOS 17.4
- Browser Safari

**Comment by excalith** [comment]
Can you please share your `settings.json` file if not private? Seems like the blurring effect kicks in differently. Will try it on my old trusty mac. Is the browser on the top is a customized Chrome or an app that runs on Chromium?

**Comment by mureev** [comment]
yes, its was Chromium based browser (Arc), but no worries. I've updated screenshot to include one from Chrome.

my config is attached, but you can also reproduce it with [online version you have](https://excalith-start-page.vercel.app). whatever works best for you.
[settings.json](https://github.com/excalith/excalith-start-page/files/14735845/settings.json)

![2024-03-24 в 18 01 03](https://github.com/excalith/excalith-start-page/assets/277148/06c1c7cd-708d-4d2c-a8ed-448619861ed3)

**Comment by excalith** [comment]
Hey @mureev, I was able to reproduce that issue on MacOS using Safari. Turns out the issue is related to how Safari handles SVG files, where it requires additional SVG definitions to handle it correctly. In this case, this is not a bug and can be fixed by changing the wallpaper asset url into a PNG file instead. If you liked the wallpaper, you might be able to find the same (GNOME wallpaper) as PNG on the internet and replace it with the SVG file in the project.

I might change the SVG file into PNG in this repository, but giving its big size and possible changes later on, it is best to switch Git LFS. Doing so will require other users to use LFS as well, and it is not convinient for most of the users.

Let me know if this answers your question. 
Cheers!
